packages/ember-engines/node_modules/.bin/_mocha

```diff
@@ -0,0 +1 @@
+../../../../node_modules/mocha/bin/_mocha
```

Comment:
I don't grok why this is here, can you remove the checked in `packages/ember-engines/node_modules` folder?